jicoco/src/main/kotlin/org/jitsi/rest/JettyBundleActivatorConfig.kt

```diff
@@ -87,5 +87,15 @@ class JettyBundleActivatorConfig(
         "default" { 8443 }
     }
 
-    override fun toString() = "host=$host, port=$port, tlsPort=$tlsPort, isTls=$isTls, keyStorePath=$keyStorePath"
+    /**
+     * Whether Jetty server version should be sent in HTTP responses
+     */
+    val sendServerVersion: Boolean by config {
+        "$legacyPropertyPrefix.jetty.sendServerVersion".from(JitsiConfig.legacyConfig)
```

Comment:
You don't need to handle it being in the legacy config, since this is a new value it will only be supported in the new config.